test/integration/hooks.spec.js

```diff
@@ -53,7 +53,7 @@ describe('hooks', function() {
   it('can fail a test in an afterEach hook', function(done) {
     helpers.runMochaJSON(
       'hooks/afterEach-hook-conditionally-fail.fixture.js',
-      args,
+      [],
```

Comment:
this was running the `dot` reporter.  that doesn't make sense with `runMochaJSON()`; it *does* make sense with `runMocha()`.